setupinfo.py

```diff
@@ -5,7 +5,7 @@
 import subprocess
 from distutils.core import Extension
 from distutils.errors import CompileError, DistutilsOptionError
-from distutils.command.build_ext import build_ext as _build_ext
+from setuptools.command.build_ext import build_ext as _build_ext
```

Comment:
Probably ok to change this since we're importing `setuptools` anyway. What was the reason why this was necessary?

Reply:
setuptools has an additional override in its `build_ext.get_ext_filename` to set the right suffix when compiling (in this case, `.cp310-win_arm64.pyd`) that distutils does not. Under normal circumstances, the presence of setuptools is enough to get the right command, so it's transparent, but when you explicitly import it from distutils (even with the setuptools bundled version of distutils) and subclass, then you override it.

Eventually setuptools will merge the two, and I think they consider imports directly from distutils to be deprecated (at least, I hope they do), so those will eventually stop working. But for the transition, they're keeping them working.

Reply:
Ok. I think we should then import setuptools before `from distutils.core import Extension`, to mark the dependency. Otherwise, this would stop working if the import order in `setup.py` changed.